src/components/common/Toast/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { View } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { useToast } from 'react-native-toast-notifications';
+
+import { isIos } from '@/constants/platform';
+import { Colors } from '@/constants/theme';
+import Close from '@/icons/material/Close.svg';
+import ErrorIcon from '@/icons/svg/ErrorIcon.svg';
+import InfoIcon from '@/icons/svg/InfoIcon.svg';
+import SuccessIcon from '@/icons/svg/SuccessIcon.svg';
+
+import Text from '../Text';
+import Touchable from '../Touchable';
+import styles from './styles';
+
+export enum ToastTypes {
+  success = 'success',
+  error = 'error',
+  info = 'info',
+}
+
+export interface ToastProps {
+  title: string;
+  message: string;
+  type: 'success' | 'error' | 'info';
+  id: string;
+}
+
+export const toastProps = {
```

Comment:
we could change this object name, to differentiate it from the interface above

Reply:
Sure!